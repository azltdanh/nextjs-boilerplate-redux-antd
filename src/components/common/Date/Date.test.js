import Date from './Date';

describe(Date.name, () => {
  it('renders', () => {
    const wrapper = shallow(<Date dateString="2020-12-31" />);
    expect(wrapper).toMatchSnapshot();
  });
});
