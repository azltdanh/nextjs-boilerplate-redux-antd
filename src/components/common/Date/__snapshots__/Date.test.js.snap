// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date renders 1`] = `
<time
  dateTime="2020-12-31"
>
  December 31, 2020
</time>
`;
